_includes/nav.html

```diff
@@ -1,46 +1,153 @@
+  </ul>
 </li>
+
 <li class="dropdown">
   <a href="#" data-toggle="dropdown" class="dropdown-toggle">
-  <span>Resources</span>
-  <b class="caret"></b>
+    <span>Community</span>
+    <b class="caret"></b>
   </a>
   <ul class="dropdown-menu">
-    <li><a href="//jakarta.ee/resources#surveys">Surveys</a> </li>
-    <li><a href="//jakarta.ee/resources#infographics">Infographics</a> </li>
-    <li><a href="//jakarta.ee/resources#datasheets">Datasheets</a> </li>
-    <li><a href="//jakarta.ee/resources#presentations">Presentations</a> </li>
-    <li><a href="//jakarta.ee/community/adopt-a-spec/">Adopt A Spec</a> </li>
-    <li><a href="//jakarta.ee/committees">Committees</a> </li>
-    <li><a href="//jakarta.ee/newsletter/">Newsletter Archives</a> </li>
-    <li><a href="//jakarta.ee/meeting_minutes/">Meeting Minutes</a> </li>
-    <li><a href="https://Jakartablogs.ee">Blog</a> </li>
-    <li><a href="//jakarta.ee/resources#meetups">Tech Talk Meet-ups</a> </li>
-    <li><a href="https://www.youtube.com/channel/UC4M7h5b6elu9VlzjLfzuXyg">Jakarta EE YouTube Channel</a> </li>
-    <li><a href="https://projects.eclipse.org/list-of-projects?combine=jakarta&amp;field_project_techology_types_tid=1102&amp;field_state_value_2=All">Jakarta EE Specification Projects</a> </li>
-    <li><a href="https://drive.google.com/open?id=1kJFh6EIpOyAWP3JkymYZXOK311WM2HXm">Jakarta EE Community Folder</a> </li>
-    <li><a href="//jakarta.ee/resources#calendar">Jakarta EE Community Calendar</a> </li>
-    <li><a href="https://start.jakarta.ee/">Starter for Jakarta EE</a> </li>
-    <li><a href="https://jakartaone.org/">JakartaOne Livestream</a> </li>
+    <li><a href="/community/">Overview</a> </li>
+    <li><a href="/community/adopt-a-spec/">Adopt A Spec</a> </li>
+    <li><a href="/blogs">Blog</a> </li>
+    <li><a href="/community/events/">Events</a> </li>
+    <li><a href="/community/get-involved">Get Involved</a> </li>
+    <li><a href="https://jakartaone.org">JakartaOne Livestream</a> </li>
+    <li><a href="/connect/">Stay Connected</a> </li>
+    <li><a href="/community/#meetups">Tech Talk Meet-ups</a> </li>
+    <li><a href="/community/videos">Videos</a> </li>
   </ul>
 </li>
-<li>
-  <a class="" href="//jakarta.ee/compatibility/">
-  Compatible Products
+
+<li class="dropdown">
+  <a href="#" data-toggle="dropdown" class="dropdown-toggle">
+    <span>Membership</span>
+    <b class="caret"></b>
   </a>
+  <ul class="dropdown-menu">
+    <li><a href="/membership">Join</a> </li>
+    <li><a href="/membership/members/">Members</a> </li>
+  </ul>
+</li>
+<li class="dropdown">
+  <a href="#" data-toggle="dropdown" class="dropdown-toggle">
+    <span>Resources</span>
+    <b class="caret"></b>
+  </a>
+  <ul class="dropdown-menu">
+    <li><a href="/resources">Overview</a> </li>
+    <li><a href="/resources#datasheets">Datasheets</a> </li>
+    <li><a href="/resources#documentation">Documentation</a> </li>
```

Comment:
Same, here. I won't post this comment on every instance but make sure to include the full URL for these type of links.